.changeset/wise-pants-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
+---
+
+We will get message on slack with reminder - if core automation tests failed, we need to create cloud-deployments release PR manually after retesting
```

Comment:
Changelog entry doesn't seem related? Was it added accidentally to the wrong PR? It looks like a feature addition whereas it's a coding mistake/typo